### DISCUSSION TITLE
Feature Request - Compare Requests

### DISCUSSION THREAD

**Discussion by PaulBroomfield** [MisterPhilip/omnibug#242] [Ideas]
Hi,

Love Omnibug!

It would be awesome if we could compare requests

e.g. I'm working on a site that has Launch installed, I trigger a Page View, make changes in the TMS, refresh the page and I would like to compare the last Page View with the new one to make sure I haven't broken anything.

Bonus feature request, being able to record a journey and replay it after my change and compare the network requests.

Thanks!

**Comment by MisterPhilip** [comment]
I really like this idea! I'm going to give it some thought on the UI and the flow for it, but I'll definitely add it to the wish list!

**Reply by PaulBroomfield** [reply]
Awesome, thank you! Happy to help you test it :)
